### PR TITLE
New credentials injection method 

### PR DESCRIPTION
**Pull request by kikettas**
New credentials injection method in Example app.

### How to test
Add the environment variables and check that GVL Example app analyze documents.

### Merging
Automatic.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -53,6 +53,11 @@ pipeline {
         sh 'rm -rf build'
         sh 'rm Example/Release-keys.xcconfig'
       }
+      post {
+        always {
+          sh 'rm -rf Example/Keys.plist || true'
```
Comment (nsobadjiev) [inline Jenkinsfile]: You don't need the `-r`, maybe not even the `-f`. It's not a directory so it's not recursive and I doubt you need to force it.

File: Example/README.md [hunk 2]
```diff
@@ -4,4 +4,4 @@ The Swift example app provides you with a sample usage of the Gini Vision Librar
 
 The Gini API SDK for iOS is used for analyzing documents and sending feedback.
 
-Before analyzing documents, you need to set your Gini API Client Id and Secret in the `ENV-Keys.Example.xcconfig` and rename it according to the configurations you want to use.
\ No newline at end of file
+Before analyzing documents, you need to set your Gini API Client ID and Client Password in your Environment Variables (`Xcode -> Product -> Scheme -> Edit scheme -> Environment Variables)` as *client_id* and *client_password*, respectively.
```
Comment (nsobadjiev) [inline Example/README.md]: Why aren't you also explaining how people can use the keys file?

File: Example/Keys.plist [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nsobadjiev) [inline Example/Keys.plist]: There's no key here. It's credentials, or client secrets. The file name is misleading.

File: Example/GiniVision/SelectAPIViewController.swift [hunk 4]
```diff
@@ -37,8 +37,7 @@ final class SelectAPIViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        let customClientId = UserDefaults.standard.string(forKey: kSettingsGiniSDKClientIdKey) ?? ""
-        let clientId = customClientId != "" ? customClientId : kGiniClientId
+        let clientId = ProcessInfo.processInfo.environment["client_id"] ?? ""
```
Comment (nsobadjiev) [inline Example/GiniVision/SelectAPIViewController.swift]: What about the file? Why are you only reading from the ENV?

File: Example/GiniVision/DocumentService.swift [hunk 5]
```diff
@@ -82,14 +93,6 @@ final class DocumentService {
         print("Gini Vision Library for iOS (\(GiniVision.versionString)) / Client id: \(clientId)")
     }
     
-    func populateSettingsPage() {
```
Comment (nsobadjiev) [inline Example/GiniVision/DocumentService.swift]: No settings page anymore?

File: .gitignore [hunk 6]
```diff
@@ -19,6 +19,11 @@ DerivedData
 *.hmap
 *.ipa
 .env
+*.xcodeproj/*
+!*.xcodeproj/project.pbxproj
+!*.xcodeproj/xcshareddata/
+!*.xcworkspace/contents.xcworkspacedata
+/*.gcno
```
Comment (nsobadjiev) [inline .gitignore]: Don't you need an ignore entry for the keys file?

Comment (kikettas) [reply .gitignore]: No, it is empty and I think it should be there.

File: Example/GiniVision/DocumentService.swift (hunk 5 above)
Comment (kikettas) [reply Example/GiniVision/DocumentService.swift]: No one was using it, it wasn't even documented. And adding a password in the settings is not the best way I think.

File: .gitignore (hunk 6 above)
Comment (nsobadjiev) [reply .gitignore]: What if I decide to use it, fill it with valid credentials and commit without realising it contains sensitive data?

File: Example/GiniVision/SelectAPIViewController.swift (hunk 4 above)
Comment (kikettas) [reply Example/GiniVision/SelectAPIViewController.swift]: Yes, I forgot about this one.

File: .gitignore (hunk 6 above)
Comment (kikettas) [reply .gitignore]: Yes, you are right. 👍

**Pull request merged**